src/js/constants/ServiceSchema.js

```diff
@@ -45,7 +45,7 @@ let SERVICE_SCHEMA = {
           type: 'number',
           default: 1,
           getter: function (service) {
-            return (service.getInstancesCount() || 0);
+            return (service.getInstancesCount() || 0) + '';
```

Comment:
In fact I just saw this on master when opening the dialog: `tracker.js:10 Warning: Failed propType: Invalid prop `startValue` of type `number` supplied to `FieldInput`, expected `string`. Check the render method of `FormControl`.`